pootle/templates/admin/language_team.html

```diff
@@ -0,0 +1,132 @@
+	<div style="background: #eee; margin: 1em; padding: 1em">
+	  <h3>Member admin</h3>
+	  <div style="padding: 1em; text-align: center">
+	    <div style="padding: 1em; text-align: left">
+	      <h4>{{form.new_member.label }}</h4>
+	      {{ form.new_member }}
+	    </div>
+	    <div style="padding: 1em; text-align: left">
+	      <h4>{{form.role.label }}</h4>
+	      {{ form.role }}
+	    </div>
+	    <div style="padding: 1em; text-align: center">
+	      <input type="submit" class="btn" value="Add team member">
+	    </div>
+	  </div>
+	</div>
+	<div style="background: #eee; margin: 1em; padding: 1em">
+	  <h4>Admins</h4>
+	  <div style="padding: 1em">
+	    <table>
+	      {{ form.admins }}
+	    </table>
+	    <div style="padding: 1em; text-align: center">
+	      <input
+		 type="submit"
+		 class="btn"
+		 name="action"
+		 value="Remove selected" />
+	    </div>
+	  </div>
+	</div>
+	<div style="background: #eee; margin: 1em; padding: 1em">
+	  <h4>Reviewers</h4>
+	  <div style="padding: 1em">
+	    <table>
+	      {{ form.reviewers }}
+	    </table>
+	    <div style="padding: 1em; text-align: center">
+	      <input
+		 type="submit"
+		 class="btn"
+		 name="action"
+		 value="Remove selected" />
+	    </div>
+	  </div>
+	</div>
+	<div style="background: #eee; margin: 1em; padding: 1em">
+	  <h4>Submitters</h4>
+	  <div style="padding: 1em">
+	    <table>
+	      {{ form.submitters }}
+	    </table>
+	    <div style="padding: 1em; text-align: center">
+	      <input
+		 type="submit"
+		 class="btn"
+		 name="action"
+		 value="Remove selected" />
+	    </div>
+	  </div>
+	</div>
+	<div style="background: #eee; margin: 1em; padding: 1em">
+	  <h4>Members</h4>
+	  <div style="padding: 1em">
+	    <table>
+	      {{ form.members }}
+	    </table>
+	    <div style="padding: 1em; text-align: center">
+	      <input
+		 type="submit"
+		 class="btn"
+		 name="action"
+		 value="Remove selected" />
+	    </div>
+	  </div>
+	</div>
+      </div>
+      <div style="float: left; width: 50%">
+	<div style="background: #eee; margin: 1em; padding: 1em">
+	  <h3>Projects</h3>
+	  <div style="padding: 1em">
+	    <ul>
+	      {% for tp in tps %}
+	      <li>{{ tp.project.fullname }}</li>
+	      {% endfor %}
+	    </ul>
+	  </div>
+	</div>
+	
+	<div style="background: #eee; margin: 1em; padding: 1em">
+	  <h3>Unit admin</h3>
+	  <div style="padding: 1em">
+	    <ul>
+	      <li>Critical: {{ stats.critical }}</li>
+	      <li>Total words: {{ stats.total }}</li>
+	      <li>Translated: {{ stats.translated }}</li>
+	      <li>Translated: {{ stats.fuzzy }}</li>
+	    </ul>
+	  </div>
+	</div>
+	<div style="background: #eee; margin: 1em; padding: 1em">
+	  <h3>Suggestion admin</h3>
+	  <div style="padding: 1em">
+	    <p>
+	      There are {{ suggestions.count }} suggestions pending
+	    </p>
+	  </div>
+	</div>
+      </div>
+    </div>
+  </form>
+</div>
+{% endblock %}
+{% endblock %}
+
+{% block breadcrumbs %}
```

Comment:
Actually there should be no breadcrumbs for this page as the dropdowns would suffice.